charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -101,3 +101,5 @@ serviceAccount:
     create: true
     name: ebs-snapshot-controller
     annotations: {}
+  node:
+    name: "" # Name of the service-account to be used
```

Comment:
Can we do create, name, annotations fields instead? Although I like the brevity, we should be consistent with the other two.

Reply:
Yes, that's true. I've updated to keep it consistent. Thanks